docs/source/user/conversion.rst

```diff
@@ -0,0 +1,97 @@
+.. _conversion:
+
+===============
+Data Conversion
+===============
+
+By default PyMySQL will convert MySQL data types into native Python types. The
+full list of conversions is:
+
+=============== =============================================
+MySQL type      Python type
+=============== =============================================
+``BIT``         [No conversion]
+``TINY``        ``Int``
+``SHORT``       ``Int``
+``LONG``        ``Int``
+``FLOAT``       ``Float``
+``DOUBLE``      ``Float``
+``LONGLONG``    ``Int``
+``INT24``       ``Int``
+``YEAR``        ``Int``
+``TIMESTAMP``   ``datetime.datetime`` or ``datetime.date`` *
+``DATETIME``    ``datetime.datetime`` *
+``TIME``        ``datetime.timedelta`` *
+``DATE``        ``datetime.date`` *
+``SET``         ``Set``
+``BLOB``        ``String``
+``TINY_BLOB``   ``String``
+``MEDIUM_BLOB`` ``String``
+``LONG_BLOB``   ``String``
+``STRING``      ``String``
+``VAR_STRING``  ``String``
+``VARCHAR``     ``String``
+``DECIMAL``     ``Decimal``
+``NEWDECIMAL``  ``Decimal``
+=============== =============================================
+
+Items marked * are returned as ``None`` if they are of an invalid format or are
+invalid dates. For example, these valid ``DATETIME`` values::
```

Comment:
I think it's useful to mention the issue with invalid dates not being converted to native python date/time types, as it took me a while to figure out what was happening when this happened to me.

Instead of removing this section we could say that this behaviour is likely to change in the future, to return strings (which sounds a good idea) rather than None?